tests/test_models/test_inference/test_forecast.py

```diff
@@ -231,6 +233,18 @@ def test_forecast_in_sample_full_no_target_failed_not_implemented_in_sample(
         ts = request.getfixturevalue(dataset_name)
         self._test_forecast_in_sample_full_no_target(ts, model, transforms)
 
+    @pytest.mark.parametrize(
+        "model, transforms, dataset_name",
+        [
+            (ChronosModel(model_name="chronos-t5-tiny", encoder_length=7), [], "example_tsds"),
+            (ChronosBoltModel(model_name="chronos-bolt-tiny", encoder_length=7), [], "example_tsds"),
+        ],
+    )
+    def test_forecast_in_sample_full_no_target_failed_chronos(self, model, transforms, dataset_name, request):
+        ts = request.getfixturevalue(dataset_name)
+        with pytest.raises(ValueError, match="Dataset doesn't have any context timestamps."):
```

Comment:
Is it really ok? Can't we do smth with this?

Reply:
I think it is ok. The common behaviour is like in other nn models, but the error is a little bit different due to ability to work with context less that `encoder_length`